code/modules/projectiles/projectile/bullets/shotgun.dm

```diff
@@ -84,16 +84,16 @@
 /obj/item/projectile/bullet/pellet/shotgun_buckshot
 	name = "buckshot pellet"
 	//skyrat edit
-	embedding = list(embed_chance=33, fall_chance=0, jostle_chance=6, ignore_throwspeed_threshold=TRUE, pain_stam_pct=1, pain_mult=6, rip_time=60)
 	damage = 7.5
-	wound_bonus = 20
+	bare_wound_bonus = 10
+	wound_falloff_tile = -2.5 // TG does this; it makes sense to do.
```

Comment:
I lowered the damage to compensate for higher wounding, so if you want to be accurate to TG, you need to buff it.

Reply:
oh wait tg actually nerfed the shotgun wtf